galaxy_importer/loaders.py

```diff
@@ -196,7 +196,7 @@ def get_requires_ansible(self):
             return
         requires_ansible = self.data.get('requires_ansible')
         if not requires_ansible:
-            return
+            raise exc.RuntimeFileError("'requires_ansible' in meta/runtime.yml is mandatory")
```

Comment:
Seems reasonable to me.

A possible alternative that honestly doesn't add much... If there were other reasons self.data['requires_ansible'] could fail maybe. But there aren't really other ways so no gain.

ie, leave your code as is unless this seems more useful 

``` python

try:
   requires_ansible = self.data['requires_ansible']
except KeyError as e:
   raise exc.RuntimeFileError("'requires_ansible' in meta/runtime.yml is mandatory") from e
```